adapter/internal/api/apis_impl.go

```diff
@@ -202,6 +220,34 @@ func updateAPI(name, version string, apiProject ProjectAPI, environments []strin
 	}
 }
 
+func extractAPIInformation(apiJsn []byte) (apiType string, apiLifeCycleStatus string,
+	productionEndpoint string, sandboxEndpoint string, apiEndpointImplementationType string, err error) {
```

Comment:
```suggestion
func extractAPIInformation(apiJsn []byte) (apiType, apiLifeCycleStatus,
	productionEndpoint, sandboxEndpoint, apiEndpointImplementationType string, err error) {
```
We can update the method signature like this, so it is shorter and also let us be cautious to not mix any because it is visible that all of them are strings.

Reply:
Fixed by https://github.com/wso2/product-microgateway/pull/1682/commits/459a83228043b61d1de8f7be99d9a054fefb1cbd